1.8/usage/service-discovery/dns-overview.md

```diff
@@ -0,0 +1,65 @@
+---
+post_title: DNS Quick Reference
+nav_title: DNS Reference
+menu_order: 20
+---
+
+DC/OS offers a lot when it comes to DNS.
+This quick reference provides a summary of the available options.
+
+# Example Setup
+
+To make things more concrete, we'll use this example application:
```

Comment:
Because there is not actual example, suggest rewrite: `To help explain concepts, this document uses a theoretical application for reference:`